### PR TITLE
Use `:where()` in CSS selectors

### PR DESCRIPTION
**Pull request by attitude**
Using `:where` should benefit our users since it does not introduce unnecessary specificity.

- - -
This change is [Reviewable](https://reviewable.io/reviews/contember/interface/501)

### CODE REVIEW

**Comment by matej21** [comment]
why not to use css layers?

**Review by attitude — COMMENTED** [review]
CSS Layers solve different issue. This is just about lowering all CSS selector specificity.

> *[Reviewable](https://reviewable.io/reviews/contember/interface/501#-:-NXAILFU3AR6-RxWUoRF:b1l6ymc)* status: 0 of 371 files reviewed, all discussions resolved (waiting on @matej21)

File: packages/ui/src/components/Layout/LayoutChrome.css [hunk 1]
```diff
@@ -3,6 +3,7 @@
   align-items: stretch;
   color: var(--cui-color--medium);
   display: flex;
+	flex: 1;
```
Comment (matej21) [inline packages/ui/src/components/Layout/LayoutChrome.css:6]: inconsistent indent. pls check all .css files to use either tabs or spaces

**Review by attitude — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/contember/interface/501)* status: 0 of 372 files reviewed, 1 unresolved discussion (waiting on @attitude and @matej21)

*[a discussion](https://reviewable.io/reviews/contember/interface/501#gh-1224053068:-NXW1knz0Y581fFCi8SY:b-896fix) (no related file):*
<details><summary><i>Previously, matej21 (David Matějka) wrote…</i></summary><blockquote>

*(Reviewable was [unable](https://github.com/Reviewable/Reviewable/issues/14) to map this GitHub inline comment thread to the right spot — sorry!)*

inconsistent indent. pls check all .css files to use either tabs or spaces
</blockquote></details>

Done.

___

**Review by matej21 — APPROVED** [review]
> Reviewed 266 of 371 files at r1, 106 of 106 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/contember/interface/501)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @attitude)

**Pull request merged**